python/pyhdk/_common.pyx

```diff
@@ -199,14 +199,17 @@ def buildConfig(*, log_dir="hdk_log", **kwargs):
   config.c_config.get().debug.log_dir = log_dir
   return config
 
-def initLogger(*, debug_logs=False, log_dir="hdk_log", **kwargs):
+def initLogger(*, debug_logs=None, log_dir="hdk_log", **kwargs):
   argv0 = "PyHDK".encode('UTF-8')
   cdef char *cargv0 = argv0
   cdef string default_log_dir = log_dir
   cdef unique_ptr[CLogOptions] opts = make_unique[CLogOptions](cargv0, default_log_dir)
   cmd_str = "".join(' --%s %r' % arg for arg in kwargs.iteritems())
   cmd_str = cmd_str.replace("_", "-")
+  if debug_logs is not None:
+    if isinstance(debug_logs, str):
+      cmd_str += ("--log-severity="+debug_logs)
+    elif debug_logs:
+      opts.get().severity_ = CSeverity.DEBUG3
```

Comment:
Did you check it's not overwritten with the default value in the following `parse_command_line` call?